Source/Diagnostics/BTDiagnostics.cpp

```diff
@@ -741,6 +741,10 @@ void BTDiagnostics::MergeBuffersForPlotfile (int i_snapshot)
 
     auto & warpx = WarpX::GetInstance();
     const amrex::Vector<int> iteration = warpx.getistep();
+    // Minimum number of digits for plotfile containing multifab data (Cell_H_XXXXX)
+    const int amrex_fabfile_min_digits = 5;
+    // Minimum number of digits for plotfile containing particle data (DATA_XXXXX)
+    const int amrex_partfile_min_digits = 5;
```

Comment:
Not sure what the "..." here would be:
```suggestion
    // number of digits for plotfile containing particle data (DATA_XXXXX)
    // the digits here are "... ids" (independent of the step) and thus always small
    const int amrex_partfile_digits = 5;
```